docs/SKUSelector.md

```diff
@@ -66,6 +66,9 @@ These are properties that you can customize in your `blocks.json` file.
 | hideImpossibleCombinations     | Boolean | If true, if a variation option leads to a combination that does not exist, that option won't appear. If false, it will appear but won't be pressable and will appear faded, will much less opacity. | `true`        |
 | maxItems                       | Number  | The maximum number of items to be displayed of a variation before showing the see more button. If the see more button should appear, it will be displayed `maxItems` - 2 options before the button  | `10`          |
 | showValueNameForImageVariation | Boolean | If true, show the name for the selected image variation to the right of the variation name. For example: `Color Red`, if the red color variation is selected.                                       | `false`       |
+| thumbnailImage | If you pass this prop it will show the first image that has this text on the image's attribute `imageText` | `undefined` |
+| visibleVariations | string[] | If you pass this array it will only display the passed names, if you pass a name that doesn't represent a variation it just doesn't show anything, which means that passing a empty array doesn't display any variation | Is `undefined`, but shows every variation |
```

Comment:
(API) I'm not sure about this default behavior. Wouldn't it be better if the default was to show nothing? It seems strange to me that the user would have to pass an empty array to represent 'show nothing' behavior.

Reply:
Yeah, but there are two problems with this:
1. It would be a breaking change because I would have to update everyone that uses the component
2. It would be weird if I have to pass a prop in order to the component to show something, because if a didn't pass anything, then there nothing rendered

Reply:
Uuhm, OK, did not notice the breaking-change hahahha makes perfect sense!